src/app/components/my-stats/my-stats.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@taiga-ui/core/styles/taiga-ui-local';
+@import "mixins";
+
+.course-islands {
+    @include island-grid();
+}
+
+.more-courses-island {
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.stats_class:hover {
+    background-color: white;
```

Comment:
dont use absolute colors. For one, it will break in the dark mode. Use taiga ui colors.